packages/panels/resources/lang/zh_CN/resources/pages/edit-record.php

```diff
@@ -2,34 +2,38 @@
 
 return [
 
-    'title' => '编辑 :label',
+    'title' => 'Edit :label',
 
-    'breadcrumb' => '编辑',
+    'breadcrumb' => 'Edit',
 
     'form' => [
 
         'actions' => [
 
             'cancel' => [
-                'label' => '取消',
+                'label' => 'Cancel',
             ],
 
             'save' => [
-                'label' => '保存',
+                'label' => 'Save changes',
             ],
 
         ],
 
```

Comment:
Why are you removing translations here? And `view-record`

Reply:
I'm sorry, this is a mistake. I forgot to deal with this `edit-record.php` file.

` view-record.php ` is changed. In order to maintain consistency in translation, I translated "view" as "详情" instead of "视图".